opm/parser/share/keywords/000_Eclipse100/O/OPERNUM

```diff
@@ -0,0 +1,5 @@
+{
+ "name" : "OPERNUM",
```

Comment:
> The parser doesn't support OPERNUM either (read the manual).

as far as I interpret this, opm-parser just does does not support/enforce the quirks/documented E100 bugs?

> Do you mean this?

yep. that's what I meant. sorry for the noise.